lib/routes/password.js

```diff
@@ -597,24 +609,28 @@ module.exports = function (
                 db.forgotPasswordVerified(passwordForgotToken)
                   .then(
                     function (accountResetToken) {
-                      return mailer.sendPasswordResetNotification(
-                        passwordForgotToken.email,
-                        {
-                          acceptLanguage: request.app.acceptLanguage,
-                          flowId: flowId,
-                          flowBeginTime: flowBeginTime
-                        }
-                      )
-                      .then(
-                        function () {
-                          return request.emitMetricsEvent('password.forgot.verify_code.completed')
-                        }
-                      )
-                      .then(
-                        function () {
-                          return accountResetToken
-                        }
-                      )
+                      return db.accountEmails(passwordForgotToken.uid)
+                        .then((emails) => {
+                          return mailer.sendPasswordResetNotification(
+                            emails,
```

Comment:
The inconsistency on these `mailer.*` methods is killing me! :-)

Some of them accept an `account` object as first argument and use the `emails` property off of it, some of them accept an `email` property on the options object, and this one appears to accept a plain list of emails.  Sooner or later this inconsistency will lead us to sending an email to somewhere we didn't intend.

Can we make all `mailer.*` methods consistently accept a list of destination addresses as first argument, to reduce footgun potential here?